cocos/ui/button.ts

```diff
@@ -628,6 +628,9 @@ export class Button extends Component {
     }
 
     public update (dt: number): void {
+        if (EDITOR) {
```

Comment:
EDITOR_NOT_IN_PREVIEW ?

Reply:
what is the different between EDITOR_NOT_IN_PREVIEW and EDITOR?

Reply:
have change EDITOR to EDITOR_NOT_IN_PREVIEW